roofit/roofitcore/src/RooRealVarSharedProperties.cxx

```diff
@@ -70,7 +71,9 @@ RooRealVarSharedProperties::RooRealVarSharedProperties(const RooRealVarSharedPro
 RooRealVarSharedProperties::~RooRealVarSharedProperties() 
 {
   if (_ownBinnings) {
-    _altBinning.Delete() ;
+    for (auto& item : _altBinning) {
+      delete item.second;
```

Comment:
Should this be an `unordered_map` with `unique_ptr` as value type?

Reply:
No, when read from a file, it doesn't own, unfortunately.